pkg/apis/example/v0alpha1/register.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 // GroupName is the group name for this API.
-const GroupName = "example.grafana.com"
+const GroupName = "example.grafana.app"
```

Comment:
I guess the Group concept is something peculiar to k8s? Any chance you share some resources about in?

Reply:
The word "group" is pretty odd... it is essentially the root path in the api structure.  Rather than use the "grafana.com" domain, we decided that "grafana.app" is insulated enough :) With this PR we now have:

<img width="580" alt="image" src="https://github.com/grafana/grafana/assets/705951/5304c5b0-7fe7-4c22-ab64-3ee302e4aa00">

and:
<img width="505" alt="image" src="https://github.com/grafana/grafana/assets/705951/acdf8b3d-fba3-49b2-8408-42dfa2a2bbeb">

Reply:
All right, this makes sense!